Tests/DatadogTests/Datadog/Mocks/DatadogTracingMocks.swift

```diff
@@ -23,8 +23,24 @@ extension TracingUUID {
     }
 }
 
+// MARK: - Integration
+
+/// `SpanOutput` recording received spans.
+class SpanOutputMock: SpanOutput {
+    struct Recorded {
```

Comment:
maybe you want to do this an array for later ...when you will assert multiple finished spans. (Maybe later :) )

Reply:
I was thinking of that 👌, but it worked well for Logging part to have only one `RecordedLog` for `LogsOutputMock`, so 🤞 should work fine for Tracing as well.